pkgs/applications/misc/solaar/default.nix

```diff
@@ -0,0 +1,47 @@
+{fetchurl, stdenv, makeWrapper, gtk3, pygobject3, pyudev}:
+let
+  version = "0.9.2";
+in
+stdenv.mkDerivation {
+  name = "solaar-${version}";
+  src = fetchurl {
+    sha256 = "0954grz2adggfzcj4df4mpr4d7qyl7w8rb4j2s0f9ymawl92i05j";
+    url = "https://github.com/pwr/Solaar/archive/${version}.tar.gz";
+  };
+
+  buildInputs =  [
+    gtk3
+    pygobject3
+    pyudev
```

Comment:
Travis is red. I was not very precise. The python packages are of course still in `python3Packages`. So just use `python3Packages.pygobject3` and `python3Packages.pyudev` and adjust the parameters on line 1.